java/src/jmri/jmrit/operations/rollingstock/cars/CarsTableModel.java

```diff
@@ -384,9 +398,8 @@ void initTable(JTable table, CarsTableFrame frame) {
     }
 
     // Cars frame table column widths, starts with Select column and ends with Edit
-    private final int[] tableColumnWidths =
-            {60, 60, 60, 65, 35, 75, 75, 75, 65, 190, 190, 140, 190, 190, 190, 65, 50, 50, 50, 50, 100,
-                    50, 100, 100, 65, 70};
+    private final int[] tableColumnWidths = { 60, 60, 60, 65, 35, 75, 75, 75, 75, 65, 190, 190, 140, 190, 190, 190, 190,
+            65, 50, 50, 50, 50, 100, 50, 100, 100, 65, 70 };
```

Comment:
It would help other developers if these columns where documented.
```
    private final int[] tableColumnWidths = {
        60, // SELECT_COLUMN
        60, // NUMBER_COLUMN
        60, // ROAD_COLUMN
        65, // TYPE_COLUMN
        35, // LENGTH_COLUMN
```
If someone wants to increase the width of the LOCATION_COLUMN, it would be much easier to find which width I want to change.